spec/support/helpers/url_helper.rb

```diff
@@ -16,7 +16,8 @@ def pkce_token_endpoint_url(options = {})
       client_id: options[:client_id]     || (options[:client] ? options[:client].uid : nil),
       redirect_uri: options[:redirect_uri]  || (options[:client] ? options[:client].redirect_uri : nil),
       grant_type: options[:grant_type]    || 'authorization_code',
-      code_verifier: options[:code_verifier]
+      code_verifier: options[:code_verifier],
+      code_challenge_method: 'plain'
```

Comment:
@rsharrott ` options[:code_challenge_method]    || 'plain',`

Reply:
On second thought, you don't need it to fallback to `plain` because the server side is supposed to fallback to it.